controllers/controllers/workloads/integration/suite_integration_test.go

```diff
@@ -110,7 +110,7 @@ var _ = BeforeSuite(func() {
 			MemoryMB:           500,
 			DefaultDiskQuotaMB: 512,
 		},
-		CFK8sControllerNamespace: "cf-k8s-controllers-system",
+		CFK8sControllerNamespace: "korifi-system",
```

Comment:
What about the stuff in `cf-k8s-api-`? Should that be korifi as well? TBH I think we should consider consolidating both of these into a single namespace, but that's probably a different story.

Reply:
Something I noticed as well.